app/globals.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
+.main.container {
+  margin-top: 5em;
+  margin-bottom: 2em;
+}
+
+.floated.right {
+  float: right;
```

Comment:
Wouldn't it be better if you used inline styles as these may give rise to conflicts?